dotCMS/src/main/java/com/dotcms/browser/BrowserQuery.java

```diff
@@ -300,6 +299,13 @@ public Builder showDotAssets(@Nonnull boolean dotAssets) {
             return this;
         }
 
+        public Builder showImages(@Nonnull List<String> mimeTypes) {
```

Comment:
show methods used to receive a boolean, such as 
```

        public Builder showImages(boolean showImages) {
        ```

The responsability of the calculation of the boolean should be outside of the showImages method